full/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
 # Adding rust-src component to nightly channel
 RUN rustup component add rust-src --toolchain $RUST_NIGHTLY_VERSION
 
+# Python packages building dependencies, can be removed afterwards
+RUN apk add -t python_build_deps python3-dev
+
+# Install ledgerctl, which is used by cargo ledger for loading or dumping an APDU app file.
+RUN pip3 install git+https://github.com/LedgerHQ/ledgerctl
+
+# Cleanup, remove packages that aren't needed anymore
+RUN apk del python_build_deps
```

Comment:
Does it really reduce the docker size?

Reply:
I don't know haha, we could keep them... we have had a habit of removing as many pip install dependencies in other images so I figured I would do the same...